pytorch_lightning/loggers/comet.py

```diff
@@ -41,14 +42,11 @@
     except ImportError:  # pragma: no-cover
```

Comment:
What can create an ImportError if comet_ml is already installed ? Possibly, let's clean this.

Reply:
there was mlflow issue which was not able to debug, so we spoke about it already https://github.com/PyTorchLightning/pytorch-lightning/pull/4860#discussion_r550866274